packages/schema/src/config/dev.ts

```diff
@@ -18,7 +18,7 @@ export default defineUntypedSchema({
      * ```
      *
      *
-     * @type {false | { key: string; cert: string }}
+     * @type {boolean | { key: string; cert: string }}
      *
      */
     https: false,
```

Comment:
This couldn’t be true after normalization. Please double check. Self generated cert is available for nuxt.options https when enabled

Reply:
Would you explain more? What do you mean?